src/components/SettingsModal.tsx

```diff
@@ -68,10 +106,19 @@ export const SettingsModal = ({ handleCloseModal, isModalOpen }: ISettingsModal)
 
   const onSave = () => {
     setSettings(localSettings);
+    const flowSettings = Object.entries(settings).filter(
+      ([key]) => key !== 'name' && key !== 'description',
+    );
```

Comment:
```suggestion
```

Reply:
We won't need this as placing the `settings` object into the `flows.metadata` one was a previous problem.